app/models/user.rb

```diff
@@ -37,7 +28,16 @@ def self.from_omniauth(auth)
     User.create!(email:, password:, first_name:, last_name: last_name.join(" "), provider: auth.provider, uid:)
   end
```

Comment:
we use the function `self.from_omniauth(auth)` to handle the Facebook authentication right?
so we create variables to save the user's attributes from the auth object ?